app/controllers/articles_controller.rb

```diff
@@ -34,17 +36,14 @@ def update
   end
 
   def destroy
+    @article = Article.find(params[:id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method